Delcam.ProductInterface.PowerMILL/Classes/PMEntitiesCollection.cs

```diff
@@ -174,6 +174,28 @@ public string GetNewEntityName(string basePrefix = null)
             return "1";
         }
 
+        /// <summary>
+        /// Draws all entities
+        /// </summary>
+        public void DrawAll()
+        {
+            System.Reflection.FieldInfo[] fi = typeof(T).GetFields(System.Reflection.BindingFlags.NonPublic | System.Reflection.BindingFlags.Static);
+            string Identifier = fi[0].GetValue(null).ToString();
```

Comment:
Small thing, can you change Identifier to identifier.  Upper case first character implies it is a Property rather than a local variable.

Reply:
Sure, i will do that.